src/store/index.js

```diff
@@ -98,6 +98,8 @@ export default new Vuex.Store({
             })
         },
         updateGameState ({commit, state, getters}) {
+            commit('updateState', getters.unitOnTimeTile ? 'time' : 'prize')
```

Comment:
I think this will be buggy in the case where we are already in the `exit` state and then move to a time tile.  We may need to add a another flag to the state to indicate that we have passed the `prize` state.  `state.game.prize` Boolean?

Reply:
good call!